upload.go

```diff
@@ -239,6 +241,10 @@ func (u Upload) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyhttp
 		}
 	}
 
+	if u.DestDirFieldName != "" {
```

Comment:
Please can you also add a check if the form field is the same as the  `dest_dir_field_name` as without that check is any `-F dir=...` possible but only `dir` is allowed from config.
Please log the case if the both values are not the same and return an `LoggableHTTPRequest()`.
I see now that should be also done for `FileFieldName` ( `file_field_name` ), will do it after your PR is commited

Reply:
Can you provide some more detailed explanation on this? I don't really get that check. Forms can have many fields. We only pick the fields specified by `FieldFileName` and `DestDirFieldName`. So to me it's self checking.

We could potentially go through the form and make sure there are no other fields? But the de facto standard is to ignore it. 

So if user configures `DestDirFieldName=foo` and the uses wrong field, eg `-F bar=...` then just ignore it / leave empty. Same as it would not be specified. The file will land on the root dir.

Reply:
never mind, I will merge it for now